App.py

```diff
@@ -2,7 +2,7 @@
 import logging  # Error handling and recording adding/editing/removing a book.
 from flask import Flask, render_template, request, redirect, url_for
 from Personal_Library_Manager import MyLibrary, Book
-
+#
 # Configure logging
 logging.basicConfig(level=logging.DEBUG)
 app = Flask(__name__)

```

Comment:
**In you add book route you need to be clear about the link input field, Try to write a descreption in the placeholder of link**